src/app/clients/ClientsPage.tsx

```diff
@@ -90,10 +93,15 @@ interface Props {
 }
 
 const ClientsPage: React.FC<Props> = (props) => {
+    const router = useRouter();
     const [selectedParcelId, setSelectedParcelId] = useState<string | null>(null);
 
     const theme = useTheme();
 
+    const onClickAddClients = (): void => {
+        router.push("/clients/add");
```

Comment:
is this better implemented as a `href` instead of an onclick which takes you to a URL?